### DISCUSSION TITLE
Убрать анимацию замены модалок на vkcom

### DISCUSSION THREAD

**Discussion by shevchux** [VKCOM/VKUI#3925] [Ideas] [answered]
vkcom довольно строгий и не имеет анимаций перехода между панелями. И кажется что анимация перехода между модалками тоже должна быть отключена. Анимацию открытия и закрытия модалки предлагаться оставить. Речь идет именно о замене одной модалки на другую в ModalRoot.

https://user-images.githubusercontent.com/22776192/209828187-27fa0159-6cd0-4e3d-b6d4-71b5e037abda.mov

Хотелось бы складывать у людей ощущение, будто они навигируются внутри одной и той же модалки. На видео выше транзишн выглядит не к месту. Больше ощущается багом.

**Comment by SevereCloud** [comment, marked as answer]
Доступно в [v5.1.0](https://github.com/VKCOM/VKUI/releases/tag/v5.1.0)
